src/main/java/org/opensearch/security/user/AccountType.java

```diff
@@ -0,0 +1,26 @@
+package org.opensearch.security.user;
+
+public enum AccountType {
+
+    INTERNAL("internal"),
+    SERVICE("service");
```

Comment:
I think this is more useful if it represents the 'user filter types', so you would include a 'no filter' value which could be the default when using `fromString(..)`